backend/infrahub/graphql/mutations/proposed_change.py

```diff
@@ -150,6 +153,8 @@ async def mutate_update(
                 raise ValidationError(str(exc)) from exc
 
         if updated_state == ProposedChangeState.MERGED:
+            if obj.is_draft.value or will_be_draft:
```

Comment:
Shouldn't this just be:

```python
if will_be_draft:
```
It could be that the PC was a draft but the mutation changes it to not be a draft in the same mutation as it's changed to the "merged" state. It looks like `will_be_draft` will always have the correct and current value?